scripts/endorser-connection.md

```diff
@@ -0,0 +1,36 @@
+To use the endorser agent in this project the following steps need to happen. Through swagger pages. 
+
+1. Generate invitation from endorser (POST `/connections/create_invitation`, receive invitation from tenant `/connections/recieve-invitation`
```

Comment:
You can create the invitation in advance and then use the `--endorser-invitation` aca-py parameter, but not sure how to set this up in docker-compose.  Possibly something to consider for the OpenShift deployment (where we can deploy/configure the Endorser agent in advance)

Reply:
will this `--endorser-invitation` parameter apply to the tenants in a multi-tenant configuration? if the a new tenant is created, will it automatically use this invitation, or will it need to prompted to use this 'default' invitation?

Reply:
I'll look into this, if the tenant doesn't auto-connect to the endorser I'll fix it ...